src/main/java/core/basesyntax/service/ReportWriterToFileImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class ReportWriterToFileImpl implements ReportWriterToFile {
+    @Override
+    public void getReport() {
+        String filePath = "src/main/resources/report.CSV";
```

Comment:
accept as an argument, to make this method more flexible, how are you gonna test it?